{% extends 'skmerApp/base.html' %}


{% block title %}<title>Skmer about</title>{% endblock title %}



{% block content %}

        <div class="container-fluid">
            <h1 class="text-left">About</h1>
            <br>
                <div class = text-justify>
                    Skmer is a fast tool for estimating distances between genomes from low-coverage sequencing reads (genome-skims), without needing any assembly or alignment step. The paper where we have described the methods and tested Skmer on simulated short reads and SRA's from previous sequencing experiments is available online (open access):
                    </p>
                    <ul>
                       <li><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1632-4">Sarmashghi, S., Bohmann, K., P. Gilbert, M. T., Bafna, V., & Mirarab, S. (2019). Skmer: assembly-free and alignment-free sample identification using genome skims. Genome Biology, 20(1), 34. https://doi.org/10.1186/s13059-019-1632-4</a></li>
                    </ul>
                    </p>Skmer is a command-line tool implemented in python. It runs <a href="http://www.genome.umd.edu/jellyfish.html">Jellyfish</a>  and <a href="http://mash.readthedocs.io/en/latest/">Mash</a> internally to efficiently compute k-mer profile of genome-skims and their intersection, and estimates the genomic distances by correcting for the effect of low coverage and sequencing error. Skmer also depends on <a href="https://github.com/lh3/seqtk">seqtk</a> for some FASTQ/A processings.
                    </p><h2>Requirements</h2>
                    The Web service requires you to upload fastq files for analysis. See <a href="{% url 'skmerResources' userId %}">Resources</a> for further details
                </div>
        </div>





{% endblock content %}
